public/app/plugins/panel/timeseries/plugins/ExemplarMarker.tsx

```diff
@@ -122,8 +146,31 @@ export const ExemplarMarker: React.FC<ExemplarMarkerProps> = ({
         {...attributes.popper}
       >
         <div className={styles.wrapper}>
+          {isLocked && (
```

Comment:
can we make this a component? I think it's a direct copy/paste from the histogram ya lazy punk